polygeist/test/polygeist-opt/sycl/argpromotion.mlir

```diff
@@ -159,7 +159,8 @@ gpu.module @device_func {
     gpu.return
   }
 
-  // COM: Test that a peelable argument can be peeled when another argument with the expected type cannot be peeled.
+  // COM:      Test that a peelable argument can be peeled when another argument with the expected type 
+  // COM-NEXT: cannot be peeled (because used in the callee by an invalid instruction).
```

Comment:
```suggestion
  // COM: Test that a peelable argument can be peeled when another argument with the expected type 
  // COM: cannot be peeled (because used in the callee by an invalid instruction).
```

Reply:
done